src/index.html

```diff
@@ -11,7 +11,179 @@
+    <form
+      class="registerForm"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="field">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              autocomplete="off"
+              name="surname"
+              id="surname"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              autocomplete="off"
+              name="name"
+              id="name"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="old">
+            How old are You?
+            <input
+              type="number"
+              id="old"
+              name="old"
+              value="12"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="date">
+            Full date of birth:
+            <input
+              type="date"
+              id="date"
+              name="birth-date"
+              placeholder="dd.mm.yyyy"
+              min="1900-01-01"
+              max="2023-04-30"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="accept-term">
+            I accept the term of the agreement
+            <input type="checkbox" id="accept-term">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="field">
+          <label for="user-email">
+            E-mail:
+            <input
+              type="email"
+              id="user-email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="field">
+          <label for="user-password">
+            Password:
+            <input
+              type="password"
+              id="user-password"
+              minlength="6"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="love-cats"
+            value="yes"
+          >
+          <label for="yes">Yes</label>
+          <input type="radio"
+                 id="no"
+                 name="love-cats"
+                 value="No"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field">
+          <label for="color">
+            What is your favorite color?
+            <input type="color" id="color">
+          </label>
```

Comment:
Add a `name` attribute to the color input element so it can be submitted with the form. For example: `name="favorite-color"`